src/pretix/base/settings.py

```diff
@@ -2146,8 +2146,9 @@ def unserialize(cls, s):
         'default': '',
         'type': str
     },
+    # here is the default setting for the updates check 
     'update_check_perform': {
-        'default': 'True',
+        'default': 'False',
```

Comment:
That looks like the correct fix, but what is all the rest for?

Reply:
that's true rest of the code i had previously commited those change then i figured this above code snippet, that's why

Reply:
THen please remove the changes if they are not necessary nor related.

Reply:
ok will remove and update the PR

Reply:
Thanks, a bit better now. Please see further comments down here.